doc/espnet2_tutorial.md

```diff
@@ -0,0 +1,172 @@
+# ESPnet2
+We are planning super major update, called `ESPnet2`.
+
+The developing status is still **under construction** yet, so please be very careful to  use with understanding following cautions:
+
+- There might be fatal bugs related to essential parts.
+- We haven't achieved comparable results to espnet1 on each tasks yet.
+
+## Main changing from ESPnet1
+
+- **Chainer free**
+  - Discarding Chainer completely.
+  - The development of Chainer is stopped at v7: https://chainer.org/announcement/2019/12/05/released-v7.html
+- **Kaldi free**
+  - It's not mandatory to compile Kaldi.
+      - **If you find some recipes requiring Kaldi mandatory, please report it. It should be dealt as a bug in ESPnet2.**
+  - We still support the features made by Kaldi optionally.
+  - We still follow Kaldi style. i.e. depending on `utils/` of Kaldi.
+- **On the fly** feature extraction & text preprocessing for training
+  - You don't need to create the features file before training, but just inputting wave data directly.
+      - We support both raw wave inputting and extracted feature.
+  - The preprocessing for Text, tokenization to characters, or sentencepieces, can be also applied in training time.
+- Discarding JSON format for describing training corpus in espnet1.
+    - Why do we discard JSON format? Because a dict object generated from large JSON file requires much memory and it also takes much times to parse such large JSON file.
```

Comment:
```suggestion
    - Why do we discard the JSON format? Because a dict object generated from a large JSON file requires much memory and it also takes much time to parse such a large JSON file.
```